keyboards/cannonkeys/bakeneko60_iso_hs/info.json

```
@@ -0,0 +1,94 @@
+{
+    "manufacturer": "CannonKeys",
+    "keyboard_name": "Bakeneko60 ISO",
+    "maintainer": "awkannan",
+    "processor": "STM32F072",
+    "bootloader": "stm32-dfu",
+    "usb": {
+        "vid": "0xCA04",
+        "pid": "0x001E",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["B1", "B0", "A7", "A5", "A4"],
+        "cols": ["B11", "B10", "B2", "A9", "A15", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "C13", "C15"]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "url": "https://cannonkeys.com",
+    "layouts": {
+        "LAYOUT": {
```

Does this follow the default `60_iso` layout? If so, that should be used. 
```suggestion
    "community_layouts": ["60_iso"]
    "layouts": {
        "LAYOUT_60_iso": {
```

---

it does-  i'll make the change in the keymaps as well